craft_store/creds.py

```diff
@@ -0,0 +1,153 @@
+"""Functions to serialize/deserialize credentials for Candid and Ubuntu One SSO."""
+
+import json
+from typing import Literal
+
+import pydantic
+from pydantic import BaseModel, Field
+
+from . import errors
+
+
+class CandidModel(BaseModel):
+    """Model for Candid credentials."""
+
+    t: Literal["macaroon"] = Field("macaroon")
```

Comment:
I believe this can be just set as `= "macaroon"`. without using Field.

Reply:
In this case I updated the attribute to have an alias, so we need `Field` anyway here (thanks for the tip though)